test/fixtures/basic/package.json

```diff
@@ -4,6 +4,8 @@
   "dependencies": {
     "run-series": "^1.1.1"
   },
+  "//": "ncp used to test https://github.com/electron-userland/electron-packager/pull/186",
+  "///": "(a module (with zero dependencies) that creates a file in node_modules/.bin)",
```

Comment:
Yes, but npm doesn't support other ways (e.g. typescript supports comments in the tsconfig.json) :(